docs/netdata-agent/versions-and-platforms.md

```diff
@@ -1,116 +1,229 @@
 
 The following distributions always provide the latest stable version of Netdata:
 
 |  Platform  | Platform Versions |    Released Packages Architecture    |
 |:----------:|:-----------------:|:------------------------------------:|
-| Arch Linux |      Latest       | All the Arch supported architectures |
-| MacOS Brew |      Latest       | All the Brew supported architectures |
+| **Arch Linux** |      Latest       | All the Arch supported architectures |
 
 ## Builds from Source
 
-We guarantee Netdata builds from source for the platforms we provide automated binary packages. These platforms are automatically checked via our CI, and fixes are always applied to allow merging new code into the nightly versions.
+We guarantee that you can build Netdata from source for the platforms where we provide automated binary packages. These platforms are automatically checked via our CI, and fixes are always applied to allow merging new code into the nightly versions.
 
-The following builds from source should usually work, although we don't regularly monitor if there are issues:
+The following builds from source should usually work for you, although we don't regularly monitor if there are issues:
 
 |              Platform               |     Platform Versions      |
 |:-----------------------------------:|:--------------------------:|
-|         Linux Distributions         | Latest unreleased versions |
-|       FreeBSD and derivatives       |         14-STABLE          |
-|       Gentoo and derivatives        |           Latest           |
-|     Arch Linux and derivatives      |      latest from AUR       |
-|                MacOS                |         13, 14, 15         |
-| Linux under Microsoft Windows (WSL) |           Latest           |
+|      **Linux Distributions**       | Latest unreleased versions |
+|    **FreeBSD and derivatives**     |         14-STABLE          |
+|     **Gentoo and derivatives**     |           Latest           |
+|   **Arch Linux and derivatives**   |      latest from AUR       |
+|             **MacOS**               |         13, 14, 15         |
 
 ## Static Builds and Unsupported Linux Versions
 
-Netdata's static builds can run on any Linux platform with supported architecture, requiring only a functioning Linux kernel of any version. These self-contained packages include everything Netdata needs to operate effectively.
+You can run Netdata's static builds on any Linux platform with supported architecture, requiring only a functioning Linux kernel of any version. These self-contained packages include everything you need for Netdata to operate effectively.
 
 ### Limitations of Static Builds
 
-Static builds lack certain features that require specific operating system support, including:
+When you use static builds, you'll miss certain features that require specific operating system support, including:
 
 - IPMI hardware sensors monitoring
 - systemd-journal functionality
```

Comment:
We can remove this, nightlies include this support since yesterday’s nightly, and regular releases will include it starting with the next minor release.